newdle/api.py

```diff
@@ -200,6 +200,17 @@ def get_my_newdles():
     return MyNewdleSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdles/in')
+def get_newdles_im_in():
+    newdle = (
+        Newdle.query.options(noload('participants'))
+        .filter(Newdle.participants.any(auth_uid=g.user['uid']))
+        .order_by(Newdle.final_dt.isnot(None), Newdle.final_dt.desc(), Newdle.id.desc())
+        .all()
+    )
+    return RestrictedNewdleSchema(many=True).jsonify(newdle)
```

Comment:
Yes, I think using the Participant would be ideal. Initially, I didn't choose it because it felt more logical to query for Newdles and return just a list of them (omitting any participants), just like "MyNewdles". But I'll test the same approach using the Participants.